### PR TITLE
make morale emoticon 3 characters and expand feline faces

### PR DESCRIPTION
**Pull request by Tsunder**

### CODE REVIEW

**Comment by Treah** [comment]
I don't think these look particularly good TBH.  Some of them are kinda cheezy

File: src/player.cpp [hunk 1]
```diff
@@ -2790,22 +2790,36 @@ void player::disp_status( WINDOW *w, WINDOW *w2 )
         col_morale = c_red;
     }
     const char *morale_str;
-    if( morale_cur >= 200 ) {
-        morale_str = "8D";
+    if( has_trait( trait_THRESH_FELINE ) ) {
+        if( morale_cur >= 200) {
+            morale_str = "@W@";
+        } else if( morale_cur >= 100 ) {
+            morale_str = "OwO";
```
Comment (Coolthulhu) [inline src/player.cpp]: someone's happy ;)

**Comment by John-Candlebury** [comment]
I dont really think this is a change for the better.

The standard emoji are well standard and get the point across better, I mean the meanings of @n@, @U@, TmT, dont really seem very clear in comparison. Also, the new ones fell like the stuff you see when people start spamming emoji.

I mean the only times I've ever I've  seen "OwO" is in parodies of  of weird sexting messages.

**Comment by Theundyingcode** [comment]
While I like this @John-Candlebury makes a good point.  People who aren't into japanese culture might have a hard time understanding a kaomoji.

**Comment by ituluwituluwzev** [comment]
(╯°Д°)╯︵ ┻━┻

**Comment by angoddu** [comment]
> I don't think these look particularly good TBH

I agree. I think they are fine as is.

**Comment by kevingranade** [comment]
Need to stash a toggle somewhere (not sure if it rates an option, maybe a mod toggle) to make it configurable, but I agree with the feadback about it making the spectrum of happiness intensity less clear, so it's a no-go for being the default.

**Comment by Coolthulhu** [comment]
I thought it was a joke, no one uses those unironically.
Right?

**Comment by AlecWhite** [comment]
>I thought it was a joke, no one uses those unironically.
Right?

I did, when I was younger.
~~nobulli~~

Also, would be fun if this could be jsonified for each post-threshold mutation.

**Comment by Theundyingcode** [comment]
I do... I took japaneses at school and as a result I loaded the japanese keyboard onto my phone/ipad... It gives me a whole menu of them.

**Comment by kevingranade** [comment]
Can go in as a mod toggle, reopen if you feel like adding it that way.

**Pull request closed without merging**